internal/kms/single-key.go

```diff
@@ -91,12 +91,15 @@ func (kms secretKey) Stat(context.Context) (Status, error) {
 	}, nil
 }
 
-func (secretKey) Metrics(ctx context.Context) (kes.Metric, error) {
-	return kes.Metric{}, errors.New("kms: metrics are not supported")
+func (secretKey) Metrics(context.Context) (kes.Metric, error) {
+	return kes.Metric{}, errors.New("kms: metrics is not supported")
```

Comment:
```suggestion
	return kes.Metric{}, errors.New("kms: metrics not supported")
```
"Metrics" is plural -> "are"... but let's just sidestep it :)